### PR TITLE
feat(content) Embersylph Adjustments

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

## Summary
This PR revisits the addition of the Embersylph from #8000 and adjusts for various reasons below.

- Now that we have the ability to define a ship's center of rotation from #9423, I've updated the Embersylph sprite to fill its whitespace used during animation and moved the center of the ship to match. I've tested the hitbox and it seems there are no problems.

- In this venue, I've also elected to use the smaller sprite (the current size of the "small" Embersylphs, half the size of the original larger ones), as my desire is for them to appear just like that - as "fairies of the wastes", rather than "a varying sized species that can be found at a much larger scale". I've dropped the base stats to what the smaller one used to match.

- As these creatures use the `swarming` personality and do not mine as the others do, they can stack up relatively quickly if you remain in a system for a decent time. My original intent with these were to be semi-rare/elusive creatures, rather than ones with similar high spawn rates as the Void Sprites or Subsidurials during the Ssil Vida activation event. For those reasons, I've slightly lowered their fleet spawn frequency from 3000 to 4000. Here are two screenshots of the change with the same interval spent in-system (though yes, this can vary):

Before:
Base Profile Screenshot 2024 01 22 - 15 49 27 36
After:
Base Profile Screenshot 2024 01 22 - 16 22 44 23

I've also thrown in a two-liner for a description, since the Embershade had one anyway.

## Save File
This save file can be used to test these changes, thanks to @warp-core:
[warp_coreprevious-830~Pre-Ssil.txt](https://github.com/endless-sky/endless-sky/files/14015705/warp_coreprevious-830.Pre-Ssil.txt)
All you have to do is fly to Ssil Vida.

## Artwork Checklist
(If any artwork was added or changed by this PR, the following must be provided.)
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/endless-sky/endless-sky-assets/pull/160
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/394

### CODE REVIEW

File: data/drak/indigenous.txt [hunk 1]
```diff
@@ -298,21 +299,7 @@ ship "Embersylph"
...
-		"frame rate" 36
-		"random start frame"
-		"scale" 0.5
-	"never disabled"
-	add attributes
-		"shields" -2800
-		"hull" -900
-		"thrust" 2
-		"turn" 10
+	description "Embersylphs are strange creatures that glow in various hues, gracefully swinging several long appendiges in a strangely methodic movement. The only known behavior these creatures are known to display is an act of curiosity for other life."
```
Comment (EjoThims) [inline data/drak/indigenous.txt:302]:
```suggestion
	description "Embersylphs are strange creatures that glow in various hues, gracefully swinging several long appendages in a strangely methodic movement. The only consistent behavior these creatures are known to display is curiosity for other life."
```

Minimum recommendation, removing redundancy and the odd "act of curiosity" phrasing in addition to spelling fix.

Comment (EjoThims) [inline data/drak/indigenous.txt:302]:
```suggestion
	description "Embersylphs are strange creatures which glow in various hues while gracefully swinging several long appendages in a strangely methodic movement. The only consistent behavior these creatures are known to display is curiosity for other life."
```

Slightly more extensive style adjustment including the changes above.

Comment (EjoThims) [inline data/drak/indigenous.txt:302]:
```suggestion
	description "Embersylphs are notably strange creatures, even compared to other known space fauna. Glowing in various hues while gracefully swinging several long appendages in a strangely methodical motion, the only consistent behavior these creatures are known to display is curiosity for other life."
```

Significantly more extensive style adjustment (and a bit more explicit comparison) including the changes above.

Could also go either way on shifting down just the "methodical motion" change to include in either of the other two options.

**Review by EjoThims — CHANGES_REQUESTED** [review]
Couple of different suggestions for the description.

**Review by EjoThims — APPROVED** [review]

**Review by OcelotWalrus — APPROVED** [review]

**Pull request merged**